src/concepts/external_data_integration.md

```diff
@@ -0,0 +1,75 @@
+
+### Environments
+
+We have a production and development environment in Fivetran.
+
+| Environment | Description                                                                                                                               | Fivetran Warehouse | Data Destination          | Access                                                                                             |
+| ----------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------------ | ------------------------- | -------------------------------------------------------------------------------------------------- |
+| Development | This environment allows developers to test their connectors during development, quickly deploy changes and make changes to imported data. | **Fivetran_Dev**   | `dev-fivetran`            | Anyone in data engineering                                                                         |
+| Production  | This environment has connectors deployed that are used in production. Access to connectors and imported data is restricted.               | **Fivetran_Prod**  | `moz-fx-data-bq-fivetran` | Only automation has access. If access to the data is required a ticket needs to be filed with SRE. |
+
+To prevent Fivetran from having access to our data warehouse, separate data destinations have been provisioned for Fivetran to dump imported data into. The data can be transformed and moved to different data destinations using bigquery-etl.
+
+When working in the **Fivetran_Dev** environment, data is written to `dev-fivetran`. All datasets are by default accessible to data engineers. So when working with sensitive data, it might be necessary to manually update the access permissions to the dataset. Ensure that the Fivetran service account has the necessary permissions to write data to the dataset when making updates.
+
+Once the development has finished, either manually delete previously created datasets or set an expiration date so that the datasets will be removed automatically after some time.
+
+## Setting up connectors
+
+This is a step-by-step guide for setting up a new connector in Fivetran, scheduling it via Airflow and transforming imported data via bigquery-etl:
+
+1. Log in to [Fivetran](https://fivetran.com) and select the environment the new connector should be created in.
+1. Click on the "Add Connector" button and search for the service data that should be imported from. If no connector is available for the service please check out ["Developing custom connectors"](#developing-custom-connectors) for writing custom connectors.
+1. Specify the name of the destination dataset Fivetran will dump data into.
+1. Configure the connector by providing API credentials and other required settings.
```

Comment:
Is it worth mentioning anything about Fivetran's default scheduling settings here? (which I think will be in effect until the last step in these instructions is completed)